### PR TITLE
Hosting on AWS amplify

### PR DESCRIPTION
**Pull request by swaminator**
PR for hosting with AWS Amplify

### CODE REVIEW

**Comment by peaceiris** [comment]
LGTM!

File: content/en/hosting-and-deployment/hosting-on-aws-amplify.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+title: Host on AWS Amplify
+linktitle: Host on AWS Amplify
+description: AWS Amplify provides the fastest way to develop and deploy a mobile or web app.
```
Comment (bep) [inline content/en/hosting-and-deployment/hosting-on-aws-amplify.md]: This is very subjective and very ... advertising. Find a more neutral sentence to describe the service.

Comment (peaceiris) [reply content/en/hosting-and-deployment/hosting-on-aws-amplify.md]: I think so. How about

```diff
- AWS Amplify provides the fastest way to develop and deploy a mobile or web app.
+ AWS Amplify is a way to develop and deploy a mobile or web app.
```

**Comment by swaminator** [comment]
@bep @peaceiris thanks for the feedback. I made the requested changes and changed the language to: "Develop and deploy a cloud-powered web app with AWS Amplify."

**Pull request merged**